### PR TITLE
ReactPanels: Adds Explore menu item

### LINKED ISSUE #19865: Logs panel: Explore shortcut `x` is not an item in panel menu
**What would you like to be added**:
**Why is this needed**:

Jumping to explore shortcut `x` is awesome, but it's not present as a panel menu item (like view/edit/share/etc.. in the 6.5.0-pre build. Adding this option, will improve the discoverability of this feature.

### PR DESCRIPTION
**Pull request by hugohaggmark**
**What this PR does / why we need it**:
This PR adds the `Explore` menuitem to all React Panels.

**Which issue(s) this PR fixes**:
Fixes #19865

**Special notes for your reviewer**:

### CODE REVIEW

**Review by torkelo — APPROVED** [review]
Nice! Great job!

**Review by davkal — APPROVED** [review]
LGTM

That code desperately needed some structure. Good work 🌟

File: public/app/core/utils/explore.ts [hunk 1]
```diff
@@ -57,13 +57,15 @@ export const lastUsedDatasourceKeyForOrgId = (orgId: number) => `${LAST_USED_DAT
...
-  timeSrv: TimeSrv
-) {
+export interface GetExploreUrlArguments {
+  panel: PanelModel;
+  panelTargets: DataQuery[];
+  panelDatasource: DataSourceApi;
+  datasourceSrv: DataSourceSrv;
+  timeSrv: TimeSrv;
+}
+export async function getExploreUrl(args: GetExploreUrlArguments) {
```
Comment (kaydelaney) [inline public/app/core/utils/explore.ts]: Could also destructure the args in the function def like

```ts
export async function getExploreUrl({ panel, panelTargets, panelDatasource, datasourceSrv, timeSrv } : GetExploreUrlArguments) {
```

but that's personal preference of course.

**Review by kaydelaney — APPROVED** [review]
LGTM! Nice work!

File: public/app/core/utils/explore.ts (hunk 1 above)
Comment (hugohaggmark) [reply public/app/core/utils/explore.ts]: Cool, forgot about that. Thanks Kay!

**Pull request merged**